tox.ini

```diff
@@ -54,6 +56,13 @@ deps = pip-tools
 commands = pip-compile --output-file requirements.txt requirements.in
 changedir = {toxinidir}
 
+[testenv:pysnooper]
+description = Ensure pip installs pysnooper
+deps = pip
+commands =
+    pip install pysnooper
+    pip show pysnooper
```

Comment:
This is not an integration test.

Think: What does this verify? Is the pysnooper code run with it? (It's not.)

Also, installing pysnooper is unneeded, and even if it were you can move it to `deps = `.